more-neat-css-things/activity-group-selector-fun.md

```diff
@@ -0,0 +1,37 @@
+# Activity: Group Selector Fun
```

Comment:
Just a note for an online group activity, we could use [codesandbox](https://codesandbox.io/) for doing the activity with online pair or swarm programming.